wb-mqtt-serial-templates/config-wbio-ai-dv-12-single.json

```diff
@@ -0,0 +1,266 @@
+{
+    "device_type": "WBIO-AI-DV-12-single", 
```

Comment:
Device type and device name are human readable. So please make it more human readable.

Reply:
device type and name are the actual device code like "WB-MR14" in every template. What else should I write here?

Reply:
again, it's for humans, not for machines. At least you can write (Single) instead of -single.